opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -306,7 +306,16 @@ def _strip_characters(ot_version):
         "cloud.account.id": "project_id",
         "host.id": "instance_id",
         "cloud.zone": "zone",
-    }
+    },
+    "gke_container": {
```

Comment:
Where does this mapping get used? (It's not obvious from the rest of the CL why these lines are needed.)

Reply:
I generalized the conversion from OT resources -> MonitoredResource in the initial resources PR.
```
return {
        "g.co/r/{}/{}".format(resource_type, gcp_resource_key,): str(
            resource.labels[ot_resource_key]
        )
        for ot_resource_key, gcp_resource_key in OT_RESOURCE_LABEL_TO_GCP[
            resource_type
        ].items()
    }
```